docs/resources/resources-glossary.md

```diff
@@ -0,0 +1,144 @@
+
+## add-in commands
+
+**Add-in commands** are UI elements, such as buttons and menus, that extend the Office UI for your add-in. When users select an add-in command element, they initiate actions such as running JavaScript code or displaying the add-in in a task pane. Add-in commands let your add-in look and feel like a part of Office, which gives users more confidence in your add-in. See [Add-in commands for Excel, PowerPoint, and Word](../design/add-in-commands.md) and [Add-in commands for Outlook](../outlook/add-in-commands-for-outlook.md) to learn more.
+
+See also: [ribbon, ribbon button](#ribbon-ribbon-button).
+
+## application
+
+**Application** refers to an Office application. The Office applications that support Office Add-ins are Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
+
+See also: [client](#client), [host](#host), [Office application, Office client](#office-application-office-client).
+
+## application-specific API
+
+Application-specific APIs provide strongly-typed objects that interact with objects that are native to a specific Office application. For example, you call the Excel JavaScript APIs for access to worksheets, ranges, tables, charts, and more. Application-specific APIs are currently available for Excel, OneNote, PowerPoint, and Word. See [Application-specific API model](../develop/application-specific-api-model.md) to learn more.
+
+See also: [Common API](#common-api).
+
+## client
+
+**Client** typically refers to an Office application. The Office applications, or clients, that support Office Add-ins are Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
+
+See also: [application](#application), [host](#host), [Office application, Office client](#office-application-office-client).
+
+## Common API
+
+Common APIs can be used to access features such as UI, dialogs, and client settings that are common across multiple Office applications. This API model uses [callbacks](https://developer.mozilla.org/docs/Glossary/Callback_function), which allow you to specify only one operation in each request sent to the Office application.
+
+Common APIs were introduced with Office 2013 and can be used to interact with Office 2013 or later. Some Common APIs, particularly the Common APIs related to Excel, are legacy APIs from the early 2010s. The application-specific APIs are preferred when possible. Other Common APIs, such as the Common APIs related to Outlook, UI, and authentication, are the modern and preferred APIs for these purposes. For details about the Common API object model, which includes APIs for interacting with Outlook, PowerPoint, and Project, see [Common JavaScript API object model](../develop/office-javascript-api-object-model.md).
```

Comment:
The callout to PowerPoint might be misleading here. WXP are in the same boat: they have application-specific APIs and also functionality from the Common APIs. Maybe it's worth mentioning that WXP have Common API functionality, but most of it has been replaced by the application-specific model?